projects/packages/blaze/README.md

```diff
@@ -11,16 +11,35 @@ Use composer to add the package to your project:
 composer add automattic/jetpack-blaze
 ```
 
-Then you can initialize it on the `admin_init` hook:
+### Initializing the features
+
+#### Using the Config Package
+
+The JITMs can be enabled using the Config package:
+
+```
+use Automattic/Jetpack/Blaze;
+
+add_action( 'plugins_loaded', 'configure_blaze', 1 );
+
+function configure_blaze() {
+    $config = new Config();
+    $config->ensure( 'blaze' );
+}
+```
```

Comment:
The `add_action` call here would be redundant because Jetpack Config already adds the init hook to run on `plugins_loaded`, can we change it to just 

```suggestion
add_action( 'plugins_loaded', 'configure_blaze', 1 );

function configure_blaze() {
    $config = new Config();
    $config->ensure( 'blaze' );
}
```
$config = new Config();
$config->ensure( 'blaze' );
```

Reply:
To get this into the Beta cut for first thing Monday (based on the timelines in this PT - pb5gDS-2fR-p2) and also following up from this comment - p8oabR-11z-p2#comment-6899 - I think we can make any necessary changes here to get this in.


In `packages/config/src/class-config.php` on line 163 we have ensure_feature( 'blaze' );, and it's got a priority of 2 ([link to pri](https://github.com/Automattic/jetpack/blob/trunk/projects/packages/config/src/class-config.php#L98)). In `packages/config/readme.md` the suggestion is to configure as low as priority 1 ([link](https://github.com/Automattic/jetpack/blob/trunk/projects/packages/config/README.md)). In that respect do you think it is it worth keeping it as is @zinigor , or is that pri difference insignificant?